cardshowviewtakenpicturesview/src/main/java/br/com/stant/libraries/cardshowviewtakenpicturesview/CardShowTakenPictureView.java

```diff
@@ -98,12 +103,23 @@ public CardShowTakenPictureView(Context context, AttributeSet attrs) {
         setupLayoutOptions();
     }
 
+    private void enableCaption(Boolean useCaption) {
+        mIsCaptionEnabled = useCaption;
+    }
+
     private void setupDialog() {
         mPreviewPicDialog                         = new Dialog(getContext(), android.R.style.Theme_Black_NoTitleBar_Fullscreen);
         mCardShowTakenPicturePreviewDialogBinding = DataBindingUtil.inflate(LayoutInflater.from(mContext),
                 R.layout.card_show_taken_picture_preview_dialog, null, false);
 
         mCardShowTakenPicturePreviewDialogBinding.setHandler(this);
+
+        if (mIsCaptionEnabled) {
+            mCardShowTakenPicturePreviewDialogBinding.cameraPhotoPreviewDialogCaptionContainer.setVisibility(View.VISIBLE);
+        } else {
+            mCardShowTakenPicturePreviewDialogBinding.cameraPhotoPreviewDialogCaptionContainer.setVisibility(View.GONE);
+        }
```

Comment:
The indentation it's strange. I don't know whether it's because it's in git or not.

Reply:
I think it is because de visual here is a little different. In the android studio is fine.